stdlib/runtime/atof/common.gr

```diff
@@ -28,106 +28,110 @@
  * IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
  * DEALINGS IN THE SOFTWARE.
 */
-
-import WasmI32 from "runtime/unsafe/wasmi32"
-import WasmI64 from "runtime/unsafe/wasmi64"
-import WasmF64 from "runtime/unsafe/wasmf64"
-import { newInt32, newInt64, newFloat64 } from "runtime/dataStructures"
-import { get_POWERS10 } from "runtime/numberUtils"
-import { get_F64_POWERS10_FAST_PATH, get_POWERS5 } from "./table"
+module Common
```

Comment:
We need to nail down the naming for these. Naming something just "Common" seems really bad

Reply:
I think it's more of a smell of the file having a bad name.

Reply:
#1637